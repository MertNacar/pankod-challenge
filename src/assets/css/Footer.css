@media only screen and (max-width: 650px) {
  img.social-image.pr-4 {
    width: 18;
    height: 18;
  }
  *.store-image {
    width: 75;
    height: 20;
  }
}
