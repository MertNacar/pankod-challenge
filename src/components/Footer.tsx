import React from "react";

const Footer: React.FC = () => {
  return (
    <footer className="myfooter d-flex bg-dark position-fixed fixed-bottom px-5">
      <div className="flex-cloumn w-100">
        <p className="text-light texts pt-5">
          Home | Terms and Conditions | Privacy Policy | Collection Statement |
          Help | Manage Account
        </p>

        <p className="text-light texts font-weight-light pt-1">
          Copyright 2016 DEMO Streaming. All Rights Reserved
        </p>

        <div className="row justify-content-between pb-5 pt-3">
          <div className="col-12 d-flex">
            <span className="mr-auto">
              <img
                className="social-image-face pr-4 pb-2"
                src="./assets/images/social/facebook-white.svg "
                alt="facebook-icon"
              />
              <img
                className="social-image pr-4 pb-2"
                src="./assets/images/social/twitter-white.svg"
                alt="twitter-icon"
              />
              <img
                className="social-image pr-4 pb-2"
                src="./assets/images/social/instagram-white.svg"
                alt="instagram-icon"
              />
            </span>

            <span className="ml-auto">
              <img
                className="store-image pr-4 pb-1"     
                src="./assets/images/store/app-store.svg"
                alt="app-store"
              />
              <img
                className="store-image pr-4 pb-1"
                src="./assets/images/store/play-store.svg"
                alt="play-store"
              />
              <img
                className="store-image pr-4 pb-1"
                src="./assets/images/store/windows-store.svg"
                alt="win-store"
              />
            </span>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
