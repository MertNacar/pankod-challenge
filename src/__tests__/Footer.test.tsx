import React from "react";
import { shallow } from "enzyme";
import Footer from "../components/Footer";

describe("Footer Component", () => {
  const wrapper = shallow(<Footer />);

  it("is exists", () => {
    expect(wrapper.exists()).toBeTruthy();
  });

  it("return Footer", () => {
    expect(wrapper).toMatchSnapshot("Footer");
  });
});
