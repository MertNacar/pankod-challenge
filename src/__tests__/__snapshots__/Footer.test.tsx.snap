// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component return Footer: Footer 1`] = `
<footer
  className="myfooter d-flex bg-dark position-fixed fixed-bottom px-5"
>
  <div
    className="flex-cloumn w-100"
  >
    <p
      className="text-light texts pt-5"
    >
      Home | Terms and Conditions | Privacy Policy | Collection Statement | Help | Manage Account
    </p>
    <p
      className="text-light texts font-weight-light pt-1"
    >
      Copyright 2016 DEMO Streaming. All Rights Reserved
    </p>
    <div
      className="row justify-content-between pb-5 pt-3"
    >
      <div
        className="col-12 d-flex"
      >
        <span
          className="mr-auto"
        >
          <img
            alt="facebook-icon"
            className="social-image-face pr-4 pb-2"
            src="./assets/images/social/facebook-white.svg "
          />
          <img
            alt="twitter-icon"
            className="social-image pr-4 pb-2"
            src="./assets/images/social/twitter-white.svg"
          />
          <img
            alt="instagram-icon"
            className="social-image pr-4 pb-2"
            src="./assets/images/social/instagram-white.svg"
          />
        </span>
        <span
          className="ml-auto"
        >
          <img
            alt="app-store"
            className="store-image pr-4 pb-1"
            src="./assets/images/store/app-store.svg"
          />
          <img
            alt="play-store"
            className="store-image pr-4 pb-1"
            src="./assets/images/store/play-store.svg"
          />
          <img
            alt="win-store"
            className="store-image pr-4 pb-1"
            src="./assets/images/store/windows-store.svg"
          />
        </span>
      </div>
    </div>
  </div>
</footer>
`;
